<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>İkiye Bölme (Bisection) Metodu</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Sans Pro</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Çöz</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>466</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_a">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>41</width>
      <height>26</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>a</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_b">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>41</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>b</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_durum">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>100</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>a - b aralığını belirleyiniz.</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
